ci-operator/jobs/openshift/openshift-ansible/openshift-openshift-ansible-release-3.10-presubmits.yaml

```diff
@@ -264,6 +264,70 @@ presubmits:
           name: prow-job-cluster-launch-e2e-openshift-ansible
         name: job-definition
     trigger: ((?m)^/test( all| e2e-aws),?(\s+|$))
+  - agent: kubernetes
+    always_run: false
+    branches:
+    - release-3.10
+    context: ci/prow/e2e-gluster
+    decorate: true
+    name: pull-ci-openshift-openshift-ansible-release-3.10-e2e-gluster
+    rerun_command: /test e2e-gluster
+    skip_cloning: true
+    spec:
+      containers:
+      - args:
+        - |
+          export RPM_REPO_OPENSHIFT_ORIGIN="$( curl -q "${RPM_REPO_BASEREF}" 2>/dev/null)"
+          ci-operator \
+            --artifact-dir=$(ARTIFACTS) \
+            --secret-dir=/usr/local/e2e-gluster-cluster-profile \
+            --template=/usr/local/e2e-gluster \
+            --target=e2e-gluster \
+            --give-pr-author-access-to-namespace=true
+        command:
+        - /bin/bash
+        - -c
+        env:
+        - name: CLUSTER_TYPE
+          value: aws
+        - name: CONFIG_SPEC
+          valueFrom:
+            configMapKeyRef:
+              key: openshift-openshift-ansible-release-3.10.yaml
+              name: ci-operator-configs
+        - name: JOB_NAME_SAFE
+          value: e2e-gluster
+        - name: RPM_REPO_BASEREF
+          value: https://storage.googleapis.com/origin-ci-test/releases/openshift/origin/release-3.10/.latest-rpms
+        - name: TEST_COMMAND
+          value: TEST_FOCUS=Suite:openshift/conformance/parallel/minimal TEST_SKIP='(\[local\]|\[templates\]|\[Feature:Router\]|Kubectl
+            logs)' run-tests
+        image: ci-operator:latest
+        name: ""
+        resources:
+          limits:
+            cpu: 500m
+          requests:
+            cpu: 10m
+        volumeMounts:
+        - mountPath: /usr/local/e2e-gluster-cluster-profile
+          name: cluster-profile
+        - mountPath: /usr/local/e2e-gluster
+          name: job-definition
+          subPath: cluster-launch-e2e-openshift-ansible.yaml
```

Comment:
Please use the ci-operator config to generate these

/cc @bbguimaraes

Reply:
`ci-operator-prowgen` would generate jobs for `prow-job-cluster-launch-e2e`, but this one needs `prow-job-cluster-launch-e2e-openshift-ansible`.

Once the new test option is added to ci-operator I'll update operator config to have these generated

Reply:
_You_ should be adding new options there :)

Reply:
Created https://github.com/openshift/ci-operator/issues/175 to track this